bcipy/helpers/triggers.py

```diff
@@ -121,197 +133,7 @@ def _calibration_trigger(experiment_clock: Clock,
-            targetness = 'fixation'
-        elif target_symbol == symbol:
-            targetness = 'target'
-        else:
-            targetness = 'nontarget'
-
-        # write to the trigger_file
-        trigger_file.write(f'{symbol} {targetness} {time}\n')
-
-    return trigger_file
-
-
-def _write_triggers_from_inquiry_free_spell(array, trigger_file):
-    """
-    Write triggers from free spell.
-
-    Helper Function to write trigger data to provided trigger_file.
-
-    It writes in the following order:
-        (I) presented letter, (II) timestamp
-    """
-
-    for i in array:
-
-        # extract the letter and timing from the array
-        (letter, time) = i
-
-        # write to trigger_file
-        trigger_file.write('%s %s' % (letter, time) + "\n")
-
-    return trigger_file
-
-
-def write_triggers_from_inquiry_icon_to_icon(inquiry_timing: List[Tuple],
-                                             trigger_file: TextIO,
-                                             target: str,
-                                             target_displayed: bool,
-                                             offset=None):
-    """
-    Write triggers from icon to icon task.
-    It writes in the following order:
-        (I) presented letter, (II) targetness, (III) timestamp
-
-    Parameters:
-    ----------
-        inquiry_timing - list of (icon, time) output from rsvp after
-            displaying a inquiry.
-        trigger_file - open file in which to write.
-        target - target for the current inquiry
-        target_displayed - whether or not the target was presented during the
-            inquiry.
-    """
-    if offset:
-        (letter, time) = inquiry_timing
-        targetness = 'offset_correction'
-        trigger_file.write('%s %s %s' % (letter, targetness, time) + "\n")
-        return
-
-    icons, _times = zip(*inquiry_timing)
-    calib_presented = 'calibration_trigger' in icons
-    calib_index = 0 if calib_presented else -1
-
-    if calib_presented:
-        target_pres_index = 1
-        fixation_index = 2
-    elif target_displayed:
-        target_pres_index = 0
-        fixation_index = 1
-    else:
-        target_pres_index = -1
-        fixation_index = 0
-
-    for i, (icon, presentation_time) in enumerate(inquiry_timing):
-        targetness = 'nontarget'
-        if i == calib_index:
-            targetness = 'calib'
-        elif i == target_pres_index:
-            targetness = 'first_pres_target'
-        elif i == fixation_index:
-            targetness = 'fixation'
-        elif icon == target:
-            targetness = 'target'
-        else:
-            targetness = 'nontarget'
-        trigger_file.write('%s %s %s' % (icon, targetness, presentation_time) +
-                           "\n")
-
-
-def trigger_decoder(mode: str, trigger_path: str = None, remove_pre_fixation=True) -> tuple:
+def trigger_decoder(trigger_path: str, remove_pre_fixation=True, offset=0, exclusion=[]) -> tuple:
```

Comment:
The diff is hard to read, but I think the documentation no longer matches the parameters.